cmd/start.go

```diff
@@ -42,9 +42,14 @@ var startServerCmd = &cobra.Command{
 	Use:   "server",
 	Short: "Start the server",
 	Run: func(cmd *cobra.Command, args []string) {
+		var err error
```

Comment:
use := instead please

Reply:
The use is with a return of a tuple, err being coupled with an already instanciated value. So this is the only way to go.